File: src/main/java/com/pigutu/app/mapper/CommentDao.java
package com.pigutu.app.mapper;


import com.pigutu.app.entity.CommentEntity;
import com.pigutu.app.entity.FeedBackEntity;
import com.pigutu.app.mapper.mybatis.BaseDao;
import com.pigutu.app.mapper.mybatis.DBMeta;
import org.apache.ibatis.annotations.Mapper;
import org.springframework.stereotype.Repository;

@Mapper
@Repository
@DBMeta(table = CommentDao.TABLE)
public interface CommentDao extends BaseDao<CommentEntity> {
    String TABLE = "comment";
}

File: src/main/java/com/pigutu/app/xforum/dao/CommentDao.kt
package com.pigutu.app.xforum.dao

import com.pigutu.app.mapper.mybatis.BaseDao
import com.pigutu.app.mapper.mybatis.DBMeta
import com.xforum.library.model.Comment
import org.apache.ibatis.annotations.Mapper
import org.springframework.stereotype.Repository

@Mapper
@Repository
@DBMeta(table = "x_comment")
interface CommentDao: BaseDao<Comment> {
}